kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue

```diff
@@ -140,20 +140,14 @@
   const resourcesNeeded = {};
   resourcesNeededShown.map(key => {
     const value = ResourcesNeededTypes[key];
-    // TODO rtibbles: remove this condition
     if (plugin_data.learnerNeeds.includes(value) || process.env.NODE_ENV !== 'production') {
```

Comment:
You removed the TODO but not the condition! 
```suggestion
    if (plugin_data.learnerNeeds.includes(value)) {
```

Reply:
updated